CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 ### Added
+- Save mixin for the `BasePredictionIntervals`. ([#87](https://github.com/etna-team/etna/pull/87))
```

Comment:
Let's write the whole name of the mixin class, we could also remove dot at the end of the line.